sample_xblocks/basic/problem.py

```diff
@@ -139,7 +155,7 @@ def student_view(self, context=None):
                         }
                     }
                     var handlerUrl = runtime.handlerUrl(element, 'check')
-                    $.post(handlerUrl, JSON.stringify(data)).success(handleCheckResults);
+                    $.post(handlerUrl, JSON.stringify(data)).success(handleCheckResults(data));
```

Comment:
I am trying to follow the logging conventions of edx-platform here, where the problem_check event has both the student answer and whether that answer is correct. The handleCheckResults is called with just which answers are correct, and does not know what the submission was, unless it is passed in. By using a closure, I have access to both.